apps/server/src/infrastructure/persistence/typeorm/repository/content/mapper/content-mapper.ts

```diff
@@ -0,0 +1,332 @@
+        return ImageContent.new(contentPayload);
+      } else {
+        return VideoContent.new(contentPayload);
+      }
+    } else if (content instanceof TypeormPost) {
+      const contentPayload: CreateContentEntityPayload<"post", "existing"> = {
+        groupId: content.groupId,
+        ownerId,
+        referred: domainReferred,
+        thumbnailRelativePath: content.thumbnailRelativePath,
+
+        id: content.id,
+        createdDateTime: content.createdDateTime,
+        updatedDateTime: content.updatedDateTime,
+        deletedDateTime: content.deletedDateTime,
+
+        numLikes: numLikes,
+        likeList: likeDomainEntityList,
+        numComments: numComments,
+        commentList: commentDomainEntityList,
+
+        title: content.title,
+        text: content.text,
+      };
+      return PostContent.new(contentPayload);
+    } else if (content instanceof TypeormBucket) {
+      const contentPayload: CreateContentEntityPayload<"bucket", "existing"> = {
+        groupId: content.groupId,
+        ownerId,
+        referred: domainReferred,
+        thumbnailRelativePath: content.thumbnailRelativePath,
+
+        id: content.id,
+        createdDateTime: content.createdDateTime,
+        updatedDateTime: content.updatedDateTime,
+        deletedDateTime: content.deletedDateTime,
+
+        numLikes: numLikes,
+        likeList: likeDomainEntityList,
+        numComments: numComments,
+        commentList: commentDomainEntityList,
+
+        title: content.title,
+        status: content.status,
+      };
+      return BucketContent.new(contentPayload);
+    } else if (content instanceof TypeormSchedule) {
+      const contentPayload: CreateContentEntityPayload<"schedule", "existing"> =
+        {
+          groupId: content.groupId,
+          ownerId,
+          referred: domainReferred,
+          thumbnailRelativePath: content.thumbnailRelativePath,
+
+          id: content.id,
+          createdDateTime: content.createdDateTime,
+          updatedDateTime: content.updatedDateTime,
+          deletedDateTime: content.deletedDateTime,
+
+          numLikes: numLikes,
+          likeList: likeDomainEntityList,
+          numComments: numComments,
+          commentList: commentDomainEntityList,
+
+          title: content.title,
+          startDateTime: content.startDateTime,
+          endDateTime: content.endDateTime,
+          isAllDay: content.isAllDay,
+        };
+      return ScheduleContent.new(contentPayload);
+    } else {
+      throw Exception.new({
+        code: Code.UTIL_PROCESS_ERROR,
+        overrideMessage: "Invalid content type.",
+      });
+    }
+  }
+
+  private static mapToOrmContentForUtil(payload: Content): TypeormContent {
+    let ormContent!: TypeormContent;
+    if (payload instanceof SystemContent) {
+      ormContent = new TypeormSystemContent();
+      (ormContent as TypeormSystemContent).text = payload.text;
+      (ormContent as TypeormSystemContent).subText = payload.subText;
```

Comment:
```suggestion
      const systemContent = new TypeormSystemContent();
      systemContent.text = payload.text;
      systemContent.subText = payload.subText;
      ormContent = systemContent;
```

요렇게 바꾸면 as 를 안써도 되긴 할 것 같은데요..
사실 큰 차이는 없어서 그냥 맘에 드는 방식으로 하셔도 될 것 같네요!